assets/js/components/notifications/BannerNotifications.js

```diff
@@ -115,7 +119,9 @@ export default function BannerNotifications() {
 			) }
 			{ ga4ModuleConnected && hasGTMScope && isGA4ModuleOwner && (
 				<Fragment>
-					<GoogleTagIDMismatchNotification />
+					{ !! hasMismatchedGoogleTagID && (
```

Comment:
I don't think we need to convert this to boolean, as the value itself is always a boolean.

Reply:
@kuasha420 You are correct, I kinda thought to double ensure it. But it doesn't seem needed. Updated